include/tuvx/radiative_transfer/solvers/delta_eddington.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2023-2024 National Center for Atmospheric Research
+// SPDX-License-Identifier: Apache-2.0
+//
+/// @file Delta-Eddington solver for radiative transfer.
+#pragma once
+
+#include <vector>
+#include <map>
+
+#include <tuvx/grid.hpp>
+#include <tuvx/profile.hpp>
+#include <tuvx/radiative_transfer/radiation_field.hpp>
+#include <tuvx/radiative_transfer/radiator.hpp>
+
+namespace tuvx {
+
+  /// @brief Radiative flux calculator that applies the delta-Eddington Approximation.
+  ///
+  /// [DEV NOTES] We can determine whether this should be a class or a set of functions
+  class DeltaEddington {
+  public:
+    /// Construct a Delta-Eddington solver.
+    DeltaEddington() = default;
+
+    /// @brief Solve the radiative transfer equation for a collection of columns
+    /// @param solar_zenith_angles Solar zenith angles for each column [radians].
+    /// @param grids Grids available for the radiative transfer calculation.
+    /// @param profiles Profiles available for the radiative transfer calculation.
+    /// @param radiation_field The calculated radiation field.
+    ///
+    /// Solves two-stream equations for multiple layers. These routines are based
+    /// on equations from: Toon et al., J.Geophys.Res., v94 (D13), Nov 20, 1989.
```

Comment:
Could we add the DOI?
https://doi.org/10.1029/JD094iD13p16287

Reply:
done!